src/main/resources/jupyter/welder-docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+services:
+  welder:
+    container_name: "${WELDER_SERVER_NAME}"
+    image: "${WELDER_DOCKER_IMAGE}"
+    network_mode: host
+    restart: always
+    environment:
+      # needed to disable auth
```

Comment:
Could you expand on this? What auth is being disabled and why?

Reply:
Not clear to me either - it was in the original branch I took this from.

Actually this whole section should be updated with the values Welder needs.